### DISCUSSION TITLE
"ANY"

### DISCUSSION THREAD

**Discussion by johngillece** [ablab/quast#167] [Q&A] [answered]
Dear Sir or Ma'am,

I assembled a genome from reads using spades and subsequently ran quast 5.0.2. I included a gff (v3) file that corresponds to the reference to which I'm mapping the contigs and also had quast call features. All seems to run well, but I'm curious why the features are type "ANY" as pasted out of the log file below as well as the circos plot that was generated.

Loaded 3239 genomic features of type "ANY". Is that indicative of something being read incorrectly?

![image](https://user-images.githubusercontent.com/6164509/73278613-1375ff00-41a9-11ea-8748-49264c4fd053.png)

**Comment by alexeigurevich** [comment, marked as answer]
Hi!
If you provide a GFF file via `--features` (or `-g`), QUAST looks for all types of genomic features specified in the GFF and uses a placeholder "ANY" in the log and in the Circos plot for the cumulative value. If you are interested in a specific type of genomic feature, e.g. CDS, you should explicitly specify this like `--features CDS:~/data/my_genome_annotation.gff`. In this case, the corresponding section of the plot will be titled "CDS". Find out more details and examples in [the manual.](http://quast.sourceforge.net/docs/manual.html#genomic_feature).

Note that if you provide gene descriptions in a plain text format or a BED file, they are considered as "genes" instead of "ANY". You may also specify a file with operons using the `--operons` option.
